src/IIIFPresentation/API/Features/Manifest/Requests/DeleteManifest.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Core;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Models.API.General;
+using Repository;
+
+namespace API.Features.Manifest.Requests;
+
+public class DeleteManifest(int customerId, string manifestId)
+    : IRequest<ResultMessage<DeleteResult, DeleteManifestType>>
+{
+    public int CustomerId { get; } = customerId;
+    public string ManifestId { get; } = manifestId;
+}
+
+public class DeleteManifestHandler(
+    PresentationContext dbContext,
+    ILogger<DeleteManifestHandler> logger)
+    : IRequestHandler<DeleteManifest, ResultMessage<DeleteResult, DeleteManifestType>>
+{
+    #region Implementation of IRequestHandler<in DeleteManifest,ResultMessage<DeleteResult,DeleteManifestType>>
+
+    public async Task<ResultMessage<DeleteResult, DeleteManifestType>> Handle(DeleteManifest request,
+        CancellationToken cancellationToken)
+    {
+        logger.LogDebug("Deleting collection {ManifestId} for customer {CustomerId}", request.ManifestId,
+            request.CustomerId);
+
+        var manifest = await dbContext.Manifests.Include(c => c.Hierarchy).FirstOrDefaultAsync(m =>
```

Comment:
Small thing but I _think_ this is basically replicating `var manifest = await dbContext.RetrieveManifestAsync();`